_layouts/welcome.html

```diff
@@ -0,0 +1,3 @@
+<h1>{{ title }}</h1>
+{{ content }} (content from the md file where welcome layout is applied)
+<button onclick="navigate -> redirect"><button>
```

Comment:
`onclick="navigate -> redirect"` ??? This is only an example of what should the button do when clicked. 

This should be replaced to something like `onclick=window.location.pathname = {{ redirect }}`
So this will replace the `window.location.pathname` with the `redirect` variable and it will cause the browser will navigate to the specified path. 

## Cheat sheet
* `window.location.pathname` - For instance here is this URL `https://github.com/zozobalogh0817/zocs/pull/67/files` this pathname will be `/zozobalogh0817/zocs/pull/67/files`
* About Jekyll and GitHub Pages
	- https://jekyllrb.com/
	- [Quick Start with GitHub pages](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll)
	- [Remote Theme On GitHub Pages](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll#adding-a-theme)